opencodeblocks/graphics/scene/clipboard.py

```diff
@@ -79,11 +79,13 @@ def _find_bbox_center(self, blocks_data):
                 ymax = y
         return (xmin + xmax) / 2, (ymin + ymax) / 2
 
-    def _deserializeData(self, data:OrderedDict):
+    def _deserializeData(self, data:OrderedDict, set_selected=True):
```

Comment:
Is there a known future use case where `set_selected` will be `False` ? Just asking out of curiosity.